### PR TITLE
user profile(bookings,announcements)

### PR DESCRIPTION
**Pull request by ChynaraMamatalieva**

### CODE REVIEW

**Comment by ChynaraMamatalieva** [comment]
user profile

File: src/main/java/kg/airbnb/airbnb/apis/UserProfileAPI.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    public UserProfileBookingResponse getUserAllBookings() {
+        return userService.getUserAllBookings();
+    }
+
+    @GetMapping("/my/announcements/")
+    public UserProfileAnnouncementResponse getAllUserAnnouncements() {
+        return userService.getAllUserAnnouncements();
+    }
+
+    @GetMapping("/announcement/find/{announcementId}")
```
Comment (beksultancs) [inline src/main/java/kg/airbnb/airbnb/apis/UserProfileAPI.java:31]: InnerPage is not your task* remove it*

**Review by beksultancs — COMMENTED** [review]
Koroche* kachan user profile alyp jatkanda*
ozunuz Information Anna announcements + bookings chyksyn
siz ozuncho endpoint kylyptyrsyz*
bir endpoint kylynyz

**Comment by ChynaraMamatalieva** [comment]
code updated

**Review by beksultancs — APPROVED** [review]
Approved*

**Pull request merged**